libraries/chain/wasm_interface.cpp

```diff
@@ -105,6 +105,8 @@ namespace eosio { namespace chain {
             once_is_enough = true;
          }
          if(cd) {
+            if (!context.is_applying_block())
+               tlog("${a} speculatively executing ${h} with eos vm oc", ("a", context.get_receiver())("h", code_hash));
```

Comment:
was this meant to be left in?

Reply:
`tlog` is trace log, but no, that was for testing.

Reply:
Sorry, yes, it is needed for the read_only_trx_test.py test. I'll add a comment about that.